esp-hal/src/aes/mod.rs

```diff
@@ -274,6 +274,7 @@ pub mod dma {
     }
 
     /// A DMA capable AES instance.
+    #[instability::unstable]
```

Comment:
Is marking the the top level module unstable not enough? Just for future me's understanding :)

Reply:
Something in the traits ends up not compiling because of some public reexport somewhere, it's a bit of a mess that I'm trying to untangle.